jenkins-integration/jenkins-jobs/scenarios/multi-pass-scenario/Jenkinsfile

```diff
@@ -4,12 +4,20 @@ node {
 }
 pipeline.multipass_pipeline([
         [job_name: 'firsttest',
+         server_version: [
+                 type: "pe",
+                 pe_version: "2016.2.0"
```

Comment:
Running the same version of PE twice probably isn't that interesting, I guess.  Maybe this should be an earlier version - but later than PE 3.3 since we have a separate `compare-3.3-to-couch` one now?  @cprice404 - WDYT?